docs/troubleshooting.rst

```diff
@@ -24,8 +24,37 @@ retaining the database::
   git submodule sync --recursive && git submodule update --init --recursive
   docker-compose pull
   docker-compose build --pull
-  docker-compose up
+  docker-compose up -d
 
+Reset a corrupt database
+------------------------
+The Kuma database can become corrupted if the system runs out of disk space,
+or is unexpectedly shutdown, and MySQL may not be able to recover it. When this
+happens, you can recover by resetting the database as well::
+
+  cd /path/to/kuma
+  docker-compose stop
```

Comment:
Good catch, I thought named volumes were excluded by ``-v``. I'll test when I can stand to lose my MySQL volume.